src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -498,6 +501,27 @@ describe("Editor Page Component", () => {
             dispatchKeyEvent("Ctrl+e");
             expect(clearRegions).toBeCalled();
         });
```

Comment:
Make sure you are up to date with the current dev. The Ctrl+e is related to the export provider and I don't know if this test is valid anymore.

Reply:
Good call. I'll make the change

Reply:
Good call. I'll make the change